docs/pipeline-configuration-files/getting-started.mdx

```diff
@@ -9,7 +9,13 @@ provisioned by Conduit at startup. It's as simple as creating a YAML file that
 defines pipelines, connectors, processors, and their corresponding
 configurations.
 
-## Example pipeline (file to file)
+## Example pipeline
+
+:::tip
+
+In our [Conduit repository](https://github.com/ConduitIO/conduit), you can find [more examples](https://github.com/ConduitIO/conduit/tree/main/examples/pipelines), but to ilustrate a simple use case we'll show a pipeline using a file as a Source, and another file as a Destination.
```

Comment:
Nitpick: the source and destination mentioned here should probably be lower-cased (methinks).

Reply:
@hariso I agree! I'll change